apps/dsp-app/src/app/user/user-form/user-form.component.html

```diff
@@ -1,92 +1,73 @@
-        <mat-option *ngFor="let lang of languagesList" [value]="lang.language"> {{ lang.value }} </mat-option>
-      </mat-select>
-    </mat-form-field>
+  <mat-form-field class="large-field">
+    <mat-select formControlName="lang" [placeholder]="'appLabels.form.user.general.language' | translate">
+      <mat-option *ngFor="let lang of languagesList" [value]="lang.language"> {{ lang.value }} </mat-option>
+    </mat-select>
+  </mat-form-field>
 
-    <div class="large-field">
-      <mat-slide-toggle formControlName="systemAdmin" *ngIf="isSysAdmin$ | async">
-        {{ 'appLabels.form.user.general.sysAdmin' | translate }}
-      </mat-slide-toggle>
-    </div>
+  <div class="large-field">
+    <mat-slide-toggle formControlName="systemAdmin" *ngIf="loggedInUserIsSysAdmin$ | async">
+      {{ 'appLabels.form.user.general.sysAdmin' | translate }}
+    </mat-slide-toggle>
+  </div>
 
-    <div class="form-panel form-action">
-      <span>
-        <button color="primary" mat-button type="reset" (click)="closeDialog.emit()">
-          {{ 'appLabels.form.action.cancel' | translate }}
-        </button>
-      </span>
-      <span class="fill-remaining-space"></span>
-      <span>
-        <!-- Update -->
-        <button mat-raised-button type="submit" color="primary" *ngIf="user" [disabled]="!userForm.valid">
-          <dasch-swiss-app-progress-indicator [color]="'white'" [status]="0" *ngIf="loading" class="submit-progress">
-          </dasch-swiss-app-progress-indicator>
-          <mat-icon *ngIf="(hasLoadingErrors$ | async) === true && !loading">close</mat-icon>
-          {{ !loading && (hasLoadingErrors$ | async) === true ? ('appLabels.form.action.retry' | translate) :
-          ('appLabels.form.action.update' | translate) }}
-        </button>
-        <!-- Create new -->
-        <button
-          mat-raised-button
-          type="submit"
-          color="primary"
-          *ngIf="!user"
-          [disabled]="!userForm.valid || (userForm.controls['password'].value.length < 8)">
-          <dasch-swiss-app-progress-indicator [color]="'white'" [status]="0" *ngIf="loading" class="submit-progress">
-          </dasch-swiss-app-progress-indicator>
-          <mat-icon *ngIf="(hasLoadingErrors$ | async) === true && !loading">close</mat-icon>
-          {{ ('appLabels.form.action.submit' | translate) }}
-        </button>
-      </span>
-    </div>
-  </form>
-</div>
+  <div class="form-panel form-action">
+    <span>
+      <button color="primary" mat-button (click)="cancel()">{{ 'appLabels.form.action.cancel' | translate }}</button>
+    </span>
+    <span class="fill-remaining-space"></span>
+    <span>
+      <!-- Update -->
+      <button
+        mat-raised-button
+        type="submit"
+        color="primary"
+        [disabled]="!userForm.valid || submitting || (!editExistingUser && userForm.controls['password'].value.length < 8)"
+        (click)="submit()">
```

Comment:
for loading spinner, should you use [appLoading] directive on button rather than progress indicator ?